soc/xtensa/intel_adsp/Kconfig

```diff
@@ -81,6 +81,9 @@ config ADSP_TRACE_SIMCALL
 	  of an enclosing simulator process.  All window contents will
 	  remain identical.
 
+config HAS_WOV_CRO
```

Comment:
Why are you introducing a Kconfig symbol for this?  Can't you just use some form of `DT_NODE_HAS_PROP` in the code?

Reply:
@galak any thoughts?

Reply:
> Am I getting the wrong inspirations?
I think so, if I am not mistaken those nordic Kconfig are there because of the HAL and there was a plan to remove them at some point. If you can use DTS macros, that would be preferred.

Reply:
Got it, will look into it.